src/features/authentication/components/LoginForm/Categories/Categories.jsx

```diff
@@ -0,0 +1,18 @@
+import "./Categories.css";
+
+const SportCard = () => {
+  return (
+    <div className="sport-card">
+      <img
```

Comment:
there also should be onclick, when i click on this category a callback function should get invoked from the parent